tests/testthat/test-lancelot.R

```diff
@@ -246,6 +254,10 @@ test_that("Test compiled lancelot components", {
     c(icu = 50),
     c(general = 50),
     c(hosp = 50),
+    c(deaths_hosp_0_49 = 1, deaths_hosp_50_54 = 1, deaths_hosp_55_59 = 1,
+      deaths_hosp_60_64 = 2, deaths_hosp_65_69 = 2, deaths_hosp_70_74 = 3,
+      deaths_hosp_75_79 = 5, deaths_hosp_80_plus = 10,
+      deaths_hosp = 30, deaths_carehomes = 10, deaths_comm = 10),
```

Comment:
Can you please separate each death stream out on its own. Same for deaths_hosp, deaths_carehomes, deaths_comm and deaths_non_hosp below please - they all should have already been on their own. The only need to pair here is when data streams pair up for binomial/beta-binomial fitting